reference/track/gtrnctr_power-kml.kml

```diff
@@ -8,10 +8,10 @@
         <end>2010-05-28T02:41:44Z</end>
       </gx:TimeSpan>
       <longitude>-122.139608</longitude>
-      <latitude>37.382794</latitude>
+      <latitude>37.382814</latitude>
```

Comment:
@viettaml , note this CL is going to change the LookAt (and thus, FlyTo) on files we handle. It's small on most common data and cases involving > hemisphere of polygons or places spanning hemispheres are subject to weird stuff already.
We can at least reduce OUR weird stuff.

Reply:
Note that the added test case in kml.cc demonstrates the existing code flying to the wrong place half way around the world.  In working a another enhancement I got so sick of GE flying to the wrong side of the world I made a detour (this PR) to fix it.

The small changes in latitude are not due to finite precisions arithmetic, they are due to the span of longitude in the bounding box which moves the midpoint along a great circle diameter towards the pole.

Also note the code currently has COMPARE_BEARING_TO_GRTCIRC defined, which matches our grtcirc calculations, but uses a spherical earth with the equatorial radius.  If we correct(undefine) that and use nvectors instead of grtcirc in the future a bunch of reference files will need to be tweaked.